engine/redis_v2/timer.go

```diff
@@ -94,11 +115,32 @@ func (t *Timer) Add(namespace, queue, jobID string, delaySecond uint32, tries ui
 	copy(buf[2:], namespace)
 	binary.LittleEndian.PutUint16(buf[2+namespaceLen:], uint16(queueLen))
 	copy(buf[2+namespaceLen+2:], queue)
-	binary.LittleEndian.PutUint16(buf[2+namespaceLen+2+queueLen:], uint16(tries))
-	binary.LittleEndian.PutUint16(buf[2+namespaceLen+2+queueLen+2:], uint16(jobIDLen))
-	copy(buf[2+namespaceLen+2+queueLen+2+2:], jobID)
+	binary.LittleEndian.PutUint16(buf[2+namespaceLen+2+queueLen:], uint16(jobIDLen))
+	copy(buf[2+namespaceLen+2+queueLen+2:], jobID)
+
+	err := t.redis.Conn.ZAdd(dummyCtx, t.Name(), &redis.Z{Score: score, Member: buf}).Err()
+	if err != nil {
+		return err
+	}
+
+	_ = t.removeFromBackup(namespace, queue, jobID)
```

Comment:
Question: what's the purpose of this operation?

Reply:
We should remove the job from backup after success to add it back to the timer set. I will update some comments about why we can ignore the error here.